### PR TITLE
fix: code-block dropdown alignment

### PR DESCRIPTION
**Pull request by zchsh**
## 🔗 Relevant links

- [Preview link][preview] 🔎
- [Asana task][task] 🎟️

## 🗒️ What

Pulls in a fix from https://github.com/hashicorp/react-components/pull/612 to adjust `<CodeTabs />` alignment when code tabs are in dropdown form _and_ there is no title for the code tabs.

## 🤷 Why

Previously, alignment was not correct - the dropdown should be left-aligned, but was right aligned.

### Before

code-tabs-before

### After

code-tabs-after

## 🧪 Testing

- [ ] Visit a page with code tabs that go to dropdown on mobile, such as [/vault/downloads][preview]
    - Ensure alignment of the dropdown matches on which side the dropdown is aligned (the dropdown should NOT extend outside the boundaries of the code block)
    - Ensure the code block is otherwise the same

## 💭 Anything else?

We never bumped to [@hashicorp/react-code-block@6.0.0](https://github.com/hashicorp/react-components/releases/tag/%40hashicorp%2Freact-code-block%406.0.0). I think this is okay, I don't believe there are any uses of the affected `onCopyCallback` prop in that release 👍

[task]: https://app.asana.com/0/1202114367927919/1202439831418533/f
[preview]: https://dev-portal-git-zsbump-code-block-hashicorp.vercel.app/vault/downloads

### CODE REVIEW

**Review by BRKalow — COMMENTED** [review]
Looks good in the preview!

File: package.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
     "@hashicorp/react-button": "^6.0.3",
     "@hashicorp/react-call-to-action": "^4.1.1",
     "@hashicorp/react-checkbox-input": "^5.0.1",
-    "@hashicorp/react-code-block": "^5.2.0",
+    "@hashicorp/react-code-block": "6.0.1-canary-20220614172337",
```
[the remark this answers is not in the document]
Comment (zchsh) [reply package.json]: ✅  Published & bumped

**Review by BRKalow — APPROVED** [review]

**Comment by zchsh** [comment]
@braydenlove Are you seeing those focus styles in a specific browser?

For context: this is one component we're still sharing between dev-dot and dot-com... and because we don't have access in dot-com to all the HDS styles and focus style class we've set up, we still use the browser default focus style.

I think this would be something we'd address when we implement a dev-dot specific code block appearance - let me know what you think! 🙇

**Comment by zchsh** [comment]
Ah okay I managed to reproduce, it seems to only show up in Chrome in _responsive mode_:

not-responsive-mode-still-blue

And in responsive mode, it seems all focus styles are orange:

responsive-mode-other-focus-also-orange

As well, on `main`, before this change was made, we still have the orange focus border:

on-main-also-orange

I think this confirms this is Chrome-specific, and perhaps even Chrome-on-mobile-specific. Going to merge this in, definitely looking forward to iterating further, and forking `code-block` in some way so that we can style it more intentionally for dev-dot 🙇‍♂️

**Comment by braydenlove** [comment]
Awesome detective work here @zchsh. Super helpful, thanks again.

**Pull request merged**